docs/themes/example-flixnet-ch4.md

```diff
@@ -33,7 +33,7 @@ Component {
 }
 ```
 
-So which image asset should we use? A game box is a rectangle with 16:9 aspect ratio, so the `banner` would be perfect for this. However, since every asset is potentially missing, we should consider showing other images and provide multiple fallbacks. If we don't have a `banner`, the next similarly sized one is the `steam` ("grid icon") asset. Because it's wider than 16:9, we'll need to crop it if we don't want black bars or squashed/scretched images (though you might prefer that). If neither image is available, I'll use `boxFront` as it tends to be commonly available.
```

Comment:
Is the `fallbacks` change necessary?
There's also a space before `stretched`.

Reply:
Fixed, it was a force-of-habit for the space after the slash!
I've changed the fallbacks back to what it was previously too :)

Reply:
Thanks!